spec

```diff
@@ -336,7 +336,7 @@ beginseg
     include "build/src/code/z_map_mark.o"
     include "build/src/code/z_moji.o"
     include "build/src/code/z_prenmi_buff.o"
-    include "build/src/code/z_msgevent.o"
+    include "build/src/code/z_nulltask.o"
```

Comment:
Anything other than `z_msgevent` will do imo since we're going to rename everything anyway. I named it this to keep it alphabetical (`z_prenmi_buff` is probably more like `z_nmi_buff`)

Reply:
rip `z_broom_wagon_task.o` 😢